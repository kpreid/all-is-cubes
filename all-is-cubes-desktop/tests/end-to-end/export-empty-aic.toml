# It is *not* an error if there is nothing to export in native format.
# (But the "empty" template has a bunch of demo stuff, so we aren't comparing it.)
bin.name = "all-is-cubes"
args = "--no-config-files --template=blank --graphics=record --output=test-export-empty.alliscubesjson"
status.code = 0

# Let the file output be caught (and ignored because there is no .out directory)
fs.sandbox = true
