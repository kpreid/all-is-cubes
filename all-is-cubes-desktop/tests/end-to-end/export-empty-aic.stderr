...

Wrote test-export-empty.alliscubesjson
